ThePensionsRegulator.Frontend/Styles/_tpr-header-menu.scss

```diff
@@ -0,0 +1,339 @@
+    display: none;
+}
+
+%mobile-menu-visible {
+    display: flex;
+    font-size: 1rem;
+    position: absolute;
+    z-index: 1000;
+    background-color: $tpr-colour-white;
+    width: 100%;
+    font-family: $govuk-font-family;
+    border-bottom: 3px solid $tpr-colour-white;
+    font-weight: $tpr-font-weight-regular;
+    flex-direction: column;
+
+    .tpr-header-search.mobile {
+        display: inline;
+        flex-direction: row;
+    }
+
+    .tpr-mobile-menu__header-search-container {
+        background: $tpr-colour-whisper;
+        align-content: center;
+        padding-left: 1rem;
+    }
+
+    .tpr-header-search.mobile .tpr-header-search__field {
+        width: 100%
+    }
+
+    .tpr-header-search.mobile .tpr-header-search-autocomplete {
+        width: 90%;
+    }
+
+    .tpr-header-search.mobile .autocomplete__wrapper {
+        position: relative;
+        width: 100%;
+    }
+
+    @media (max-width: 960px) {
+        .tpr-header-menu-nav.active .tpr-header-search.mobile .tpr-header-search__button {
+            margin-left: -37px;
+        }
+    }
+
+    @media(max-with:764px) {
+
+        .tpr-header-search.mobile .tpr-header-search {
+            display: none
+        }
+    }
+
+    ul {
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        list-style-type: none;
+        background-color: $tpr-colour-white;
+
+        li {
+            border-bottom: solid $tpr-colour-dark-grey 1px;
+        }
+    }
+
+    a {
+        text-decoration: none;
+        padding: 1rem;
+        display: flex;
+
+        &:hover {
+            text-decoration: underline;
+        }
+
+        &:focus {
+            background: $tpr-colour-tangerine-yellow;
+            border-bottom: 3px solid $tpr-colour-black;
+        }
+    }
+
+    .navigation__menu-item {
+        position: relative;
+    }
+
+    .navigation__sub-menu {
+        display: none;
+    }
+
+    .navigation__sub-menu.active {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .arrow_container {
+        padding: 4.5px;
+        position: absolute;
+        left: 85%;
+        top: 4%;
+        padding: 0.7rem;
+    }
+
+    .arrow {
+        display: inline-block;
+        border: solid #006ebc;
+        border-width: 0 3px 3px 0;
+        padding: 4.5px;
+        left: 93%;
+        top: 40%;
+        margin: 0.3rem;
+    }
+
+    .right {
+        transform: rotate(-45deg);
+        -webkit-transform: rotate(-45deg);
+        top: 27%;
+    }
+
+    .down {
+        transform: rotate(45degg);
+        -webkit-transform: rotate(45deg);
+        top: 1rem;
+    }
+
+    .arrow:focus-visible {
+        outline: $govuk-focus-width solid $govuk-focus-colour;
+    }
+
+    .navigation__sub-menu-item a {
+        font-size: 0.9rem;
+        width: 100%
+    }
+
+    .navigation__sub-menu-item-title {
+        border-bottom: solid $tpr-colour-dark-grey 1px;
+        display: flex;
+    }
+}
+
+.tpr-header-menu-nav.active {
```

Comment:
You've got a lot of CSS classes in play that do not follow the [BEM convention](https://getbem.com/introduction/). Examples include `arrow`, `right`, `navigation__sub-menu-item`, `nav-overlay`, `active`, `mobile`, `open`, `closed`. 

These are classes that might easily be defined by some other element on the page and cause this to break the other element or vice versa. All classes for the mobile menu should start with `tpr-mobile-menu`. An element inside should look something like `tpr-mobile-menu__sub-item`. A state change should look something like `tpr_mobile-menu--open`.

Reply:
All good apart from `si-glyph-fill`.